eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/MessageOnion.scala

```diff
@@ -83,31 +83,56 @@ object MessageOnion {
       if (records.get[InvoiceRequest].nonEmpty) return Left(ForbiddenTlv(UInt64(64)))
       if (records.get[Invoice].nonEmpty) return Left(ForbiddenTlv(UInt64(66)))
       if (records.get[InvoiceError].nonEmpty) return Left(ForbiddenTlv(UInt64(68)))
+      if (records.unknown.nonEmpty) return Left(ForbiddenTlv(records.unknown.head.tag))
```

Comment:
nit: probably worth a comment explaining that even odd unknown fields are an issue because they can be used to probe the identity of blinded nodes?

Reply:
Done